docs/releases/1.11.txt

```diff
@@ -446,6 +446,14 @@ Protection against insecure redirects in :mod:`django.contrib.auth` and ``i18n``
 and :func:`~django.views.i18n.set_language` protect users from being redirected
 to non-HTTPS ``next`` URLs when the app is running over HTTPS.
 
+``get_or_create`` and ``update_or_create`` check keyword arguments and ``defaults``
+-----------------------------------------------------------------------------------
+
+:meth:`~django.db.models.query.QuerySet.get_or_create` and
+:meth:`~django.db.models.query.QuerySet.update_or_create` ensure that keyword
+arguments and defaults are model fields. Keyword arguments containing a double
+underscore (`__`) will not be checked.
```

Comment:
Keyword arguments containing a `LOOKUP_SEP` won't be used for model creation, so there is no need to check them. (https://github.com/francoisfreitag/django/blob/f4bde7d9a10312414c8da33adfcad701a1463bce/django/db/models/query.py#L528)

Actually, the `get` operation that's performed by both `get_or_create` or `update_or_create` already checks lookup correctness. The only change introduced by this PR is `defaults` checking. I'm updating the release notes.